test/release/examples/primers/parIters.chpl

```diff
@@ -372,38 +351,35 @@ writeln("After reassigning A using parallel zippering and A as the leader:");
 writeln(A);
 writeln();
 
-
 //
 // Finally, as mentioned at the outset, operations that are equivalent
 // to zippering also use leader-follower iterators, so for example
-// the following whole-array assignment will use A's leader and
-// count()'s follower:
+// the following whole-array assignment will use ``A``'s leader and
+// ``count()``'s follower:
 //
 A = count(probSize, low=100);
 
 writeln("After reassigning A using whole-array assignment:");
 writeln(A);
 writeln();
 
-
 //
-// Some closing notes:
+// Closing notes
+// -------------
+// Chapel data types like records and classes can support iteration
+// by defining iterator methods (invoked by name) or ``these()`` iterators
+// which support iterating over variables of that type directly.  Such
+// iterator methods can be overloaded to support leader-follower
+// versions as well to permit parallel iteration over the variable.
 //
-//   Chapel data types like records and classes can support iteration
-//   by defining iterator methods (invoked by name) or these() iterators
-//   which support iterating over variables of that type directly.  Such
-//   iterator methods can be overloaded to support leader-follower
-//   versions as well to permit parallel iteration over the variable.
+// As mentioned at the outset, our leader-follower scheme has a number
+// of changes planned for it such as interface improvements and better
+// error checking.  We'll update this primer as we improve these features.
 //
-//   As mentioned at the outset, our leader-follower scheme has a number
-//   of changes planned for it such as interface improvements and better
-//   error checking.  We'll update this primer as we improve these features.
-//
-
-
+// Appendix:
```

Comment:
It looked rather strange to me, as it appears in the sidebar when it really doesn't matter at all. I'll pass if that's alright.